js/components/common/react_modal.js.jsx

```diff
@@ -92,17 +178,43 @@ class Description extends React.Component {
         <p>
           <strong>Timetable: </strong>
         </p>
-        {this.props.sessions.map(function (lecture, i) {
-          return (
-            <p key={i}>
-              {lecture.meetData.code +
-                lecture.meetData.session +
-                "-" +
-                lecture.meetData.section}
-            </p>
+
+        {Object.keys(this.props.sessions).map(session =>
+          this.props.sessions[session].length !== 0 ? (
+            <div key={session}>
+              <h2>{this.props.course.name + "-" + session}</h2>
```

Comment:
This heading is a little large. For now let's just use `strong`, like the other "headings" in this modal.